Index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="Style.css" />
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <div class="mobile-container">
+      <nav class="topnav">
+        <a href="#home" class="active">My Logo</a>
+
+        <a class="icon">
+          <img src="images/btn.png" />
+        </a>
+      </nav>
```

Comment:
- [ ] The container of the section should be wrapped in a `header` tag. Kindly work on it. :+1:

Reply:
you mean i should put all of this code inside a header tag? like this
<header>

      <nav class="topnav">
        <a href="#home" class="active">My Logo</a>

        <a class="icon">
          <img src="images/btn.png" />
        </a>
      </nav>
</header